### DISCUSSION TITLE
EdgeTX 2.10.1 + Flysky NV14 + TBS Crsossfire micro tx 6.19 NOT WORK!!!!!!

### DISCUSSION THREAD

**Discussion by CarqueiX** [EdgeTX/edgetx#5196] [Q&A]
Hello, I have installed the EdgeTX 2.10.1 firmware on my Flysky NV14 radio station. I use Crossfire's Micro TX module in version 6.19. I have installed AgentX Lite 0.98 and it is not able to communicate with my module.
Baudrate = 400K.
If I installed EdgeTX 2.9.0 and AgentX Lite 0.98, I do have communication between the module and the transmitter.
What can happen to the latest version??? Is there a bug with Cross???
I previously used this same station with OpenTX and cross 6.19 without any problem.
What do you think could be happening????
The telemetry is falling all the time, is this because of something about Edge??? or does it have to do with Betaflight 4.5????
Thank you all and if you can give me any ideas.

**Comment by pfeerick** [comment]
I don't usually use Crossfire, especially with NV14, so this is a very limited duration test, but 2.10.1 on NV14 with Crossfire Micro TX running FW 6.19 appears to be working fine for me... I was originally using TBS AgentX Lite 0.96, but updated to 0.98 with no change. The only time I had telemetry loss/telemetry recovered prompts was when I had TBS AgentX Lite actually running, and that is most likely due to it querying the Crossfire module or something... it wasn't very frequent. Was able to see both the module and receiver configuration in AgentX Lite. Baud rate was set to 400k... anything else didn't work. The latest beat XF firmware - 6.36 - also works at 921k baud on the NV14. Can't say much more than it is working for me... sorry I can't help more!

![image](https://github.com/EdgeTX/edgetx/assets/5500713/d1d8db24-fd0f-425b-9d93-bc50e8a56d46)
![image](https://github.com/EdgeTX/edgetx/assets/5500713/47fdf649-f240-450a-90cd-e16203cb0cb4)
![image](https://github.com/EdgeTX/edgetx/assets/5500713/1be4b86a-09f2-4efb-8b18-29e49241cd31)
![image](https://github.com/EdgeTX/edgetx/assets/5500713/0e1096cb-bf9f-4975-b38f-9ffe0a656f0c)

**Reply by j-aub** [reply]
Hi, I'm having the same problem. With EdgeTX 2.10.1 TBS agent lite doesn't find the crossfire module at all (tested all baud rates). Downgrading EdgeTX to 2.9.4 fixes everything.

**Comment by CarqueiX** [comment]
I have tried with:
2.10.1 = not working
2.10.0 = not working
2.9.0 = not working
Finally, with 2.8.0 = if it works.
TBS = 6.36
AgentX = 0.98
I have fixed, lossless telemetry.
Baud = 115k
Transmission module = one bit

Enviouse desde Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
